website/src/app/[lang]/[country]/(website)/(home)/section-6-card.tsx

```diff
@@ -0,0 +1,78 @@
+'use client';
+
+import { Card, Modal, Typography } from '@socialincome/ui';
+import Image from 'next/image';
+import { useCallback, useRef } from 'react';
+import sdgLogo from './sdgLogo.svg';
+
+type SectionCardProps = {
+	titles: {
+		main: string;
+		articles: string;
+		faqs: string;
+	};
+	items?: string[];
+	paragraphs?: string[];
+	articles?: { title: string; author: string; link: string }[];
+};
+
+export function SectionCardSix({ titles, items = [], paragraphs = [], articles = [] }: SectionCardProps) {
+	const ref = useRef<HTMLDialogElement>(null);
+	const handleShow = useCallback(() => ref.current?.showModal(), [ref]);
+
+	return (
+		<>
+			<Card normal bordered className="border-neutral my-4 cursor-pointer lg:mx-4" onClick={handleShow}>
+				<Card.Body>
+					<Card.Title>
+						<Typography size="2xl" weight="bold" color="accent">
+							{titles.main}
+						</Typography>
+					</Card.Title>
+					<div className="grid grid-cols-1 gap-4 sm:grid-cols-2 lg:grid-cols-1">
+						{items.map((item, index) => (
+							<div key={index} className="flex flex-row items-start space-x-2">
+								<Typography size="3xl" weight="bold">
+									{item}
+								</Typography>
+							</div>
+						))}
+						<Image className="h-auto max-w-xs" src={sdgLogo} alt="" />
```

Comment:
`w-auto` instead of `h-auto` will make sure the image doesn't overflow the card on smaller screens

Reply:
Also setting the `alt` parameter would be good. For example to something like "Sustainable Development Goals Logo".

Reply:
Done.